### PR TITLE
Fix capitalization in figure captions

### PR DESCRIPTION
**Pull request by apinnick**
- Change figure captions from title case to sentence case

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.10/Katello 4.12
* [x] Foreman 3.9/Katello 4.11 (planned Satellite 6.15)
* [ ] Foreman 3.8/Katello 4.10
* [ ] Foreman 3.7/Katello 4.9 (Satellite 6.14)
* [ ] Foreman 3.6/Katello 4.8
* [ ] Foreman 3.5/Katello 4.7 (Satellite 6.13; orcharhino 6.6/6.7)
* [ ] Foreman 3.4/Katello 4.6 (EL8 only)
* [ ] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only; orcharhino 6.4/6.5 on EL8 only)
* We do not accept PRs for Foreman older than 3.3.

### CODE REVIEW

File: guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc [hunk 1]
```diff
@@ -10,7 +10,7 @@ ifdef::satellite[]
 image::common/user-data-sequence-satellite.png[]
```
Comment (maximiliankolb) [inline guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc:10]: Please add alt texts line 10 and 16.

**Review by maximiliankolb — APPROVED** [review]
One small comment, rest LGTM

File: guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc (hunk 1 above)
Comment (apinnick) [reply guides/common/modules/proc_using-vmware-cloud-init-and-userdata-templates-for-provisioning.adoc:10]: Already done - https://github.com/theforeman/foreman-documentation/pull/2843

**Comment by maximiliankolb** [comment]
Merged to "master" and cherry-picked: :heavy_check_mark: 
   5552858ebb..d72efbdd66  3.10 -> 3.10
   bacbb1564b..c3898c601c  3.9 -> 3.9

**Pull request merged**